ansible/roles/debops.postfix_virtual_ldap/defaults/main.yml

```diff
@@ -0,0 +1,685 @@
+# PKI / TLS configuration [[[
+# ---------------------------
+
+# .. envvar:: postfix_virtual_ldap__pki [[[
+#
+# Enable or disable support for TLS in Postfix, managed by the :ref:`debops.pki`
+# Ansible role.
+postfix_virtual_ldap__pki: '{{ (True
+                               if (ansible_local|d() and ansible_local.pki|d() and
+                                   ansible_local.pki.enabled|d() | bool)
+                               else False) | bool }}'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__pki_path [[[
+#
+# Absolute path to the directory where PKI realms are located.
+postfix_virtual_ldap__pki_path: '{{ ansible_local.pki.path
+                                    if (ansible_local|d() and ansible_local.pki|d() and
+                                       ansible_local.pki.path|d())
+                                    else "/etc/pki/realms" }}'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__pki_realm [[[
+#
+# Name of the default PKI realm used by Postfix.
+postfix_virtual_ldap__pki_realm: '{{ ansible_local.pki.realm
+                                     if (ansible_local|d() and ansible_local.pki|d() and
+                                         ansible_local.pki.realm|d())
+                                     else "domain" }}'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__pki_ca [[[
+#
+# Name of the Root Certificate Authority certificate file used by Postfix,
+# relative to the PKI realm directory.
+postfix_virtual_ldap__pki_ca: '{{ ansible_local.pki.ca
+                                  if (ansible_local|d() and ansible_local.pki|d() and
+                                      ansible_local.pki.ca|d())
+                                  else "CA.crt" }}'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__pki_ca_crt [[[
+#
+# Filename of the root CA certificate used by the ``debops.postfix_virtual_ldap`` role
+# to verify TLS connections to the LDAP server.
+postfix_virtual_ldap__pki_ca_crt: '{{ postfix_virtual_ldap__pki_path + "/"
+                                    + postfix_virtual_ldap__pki_realm + "/"
+                                    + postfix_virtual_ldap__pki_ca }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# Secrets [[[
+# -------------------------
+
+# .. envvar:: postfix_virtual_ldap__no_log [[[
+#
+# This variable can be used in the Ansible tasks to set the ``no_log``
+# parameter. Changing it to ``False`` can help with debugging issues with the
+# roles, but otherwise it should be set to ``True`` to ensure that Ansible
+# doesn't leak sensitive data in its logs.
+postfix_virtual_ldap__no_log: True
```

Comment:
I disagree.  I think it is useful to specify this type of no_log condition ad-hoc while debugging, whitout modifying the role in the filesystem.
Example: 
`debops service/postfix_virtual_ldap  -e postfix_virtual_ldap__no_log=False -v`

Reply:
Very well, let's leave it for now then. In my opinion in production code there shouldn't be any debugging options at all to avoid accidental secret exposure, but I realize that makes actual debugging much harder on larger scales. The next best thing would be to implement some way of global, generalized debugging flag but with Ansible design the only way to get reliable global variables is either to stick them in inventory or to add a new role that's included in all other roles similarly to how `debops.secret` works. Maybe that would be the way to go - finally make a `debops.globals` role that's included everywhere via `import_role` and be done with it.